credential/cargo-credential/examples/stdout-redirected.rs

```diff
@@ -15,7 +15,7 @@ impl Credential for MyCredential {
         _args: &[&str],
     ) -> Result<CredentialResponse, Error> {
         // Informational messages should be sent on stderr.
-        eprintln!("message on stderr should be sent the the parent process");
+        eprintln!("message on stderr should be sent the parent process");
```

Comment:
I believe this is intended to be "to the parent process".

```suggestion
        eprintln!("message on stderr should be sent to the parent process");
```

Reply:
Thanks. Done